github/event_types.go

```diff
@@ -332,6 +356,56 @@ type PingEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// ProjectEvent is triggered when project is created, modified or deleted.
+// The webhook event name is "project".
+//
+// GitHub API docs https://developer.github.com/v3/activity/events/types/#projectevent
```

Comment:
Still missing a colon here, please include it for consistency.

See relevant PR #562.